rails/client/.stylelintrc

```diff
@@ -4,7 +4,7 @@
     "max-nesting-depth": 3,
     "scss/at-mixin-pattern": "^(kaiju-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$",
     "selector-class-pattern": [
-      "^kaiju-(?:[A-Z][a-zA-Z0-9]+)*(?:-[a-z0-9][a-zA-Z0-9]+)*?(?:--[a-z0-9][a-zA-Z0-9-]+)*?$|^(?:kaiju-)?(?:u|is|has|js)(?:-[a-z0-9][a-zA-Z0-9-]*)+$",
+      "(?:[a-zA-Z0-9]+)",
```

Comment:
Starting to use CSS modules, this allows any selector pattern. I couldn't find a rule that just disabled it all together.